apps/evans/pages/_document.tsx

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 import { Html, Head, Main, NextScript } from 'next/document';
+import { NextBgStaticCss } from 'next-bg-image';
 
 const Document: React.FC = () => (
   <Html lang="en">
-    <Head></Head>
+    <Head>
+      <NextBgStaticCss />
```

Comment:
this is what i mentioned in slack, for the time being we have to put this component in the dom once.